packages/volto-slate/news/6264.feature

```diff
@@ -0,0 +1 @@
+Passes the `user` to the `restricted` function of the block settings. @wesleybl
```

Comment:
I think the thingie for `user` is called a property. Please correct me if I am wrong.

```suggestion
Pass the `user` property to the `restricted` function of the block settings. @wesleybl
```

Reply:
The term `property` is usually used when we call a React component. Here we are calling a simple function.

Reply:
Updated suggestion.

Reply:
Actually `user` is an object, which is passed to the `restricted` function.

So it could be:

`Pass the `user` object to the `restricted` function of the block settings.`

Ok?

Reply:
Updated suggestion.

Reply:
Done